int32 start_submap_index
int32 end_submap_index
---
geometry_msgs/Point[] points